staging/src/k8s.io/kubectl/pkg/cmd/label/label.go

```diff
@@ -409,15 +417,21 @@ func labelFunc(obj runtime.Object, overwrite bool, resourceVersion string, label
 	}
 
 	for key, value := range labels {
+		if val, ok := objLabels[key]; !ok || (ok && val != value) {
+			added = true
```

Comment:
if `ok && val != value` is true, that means we're modifying, right? just setting added here means we will potentially only print `added`, not `modified`, right?

Reply:
if so, need a test that demonstrates this to make sure we print the right thing

Reply:
If so, I'd like to introduce one more variable `modified`:
 - `added`: only when new labels are added.
 - `removed`: only when labels are removed.
 - `modified`: when labels are replaced or both `added` and `removed` happen.

is it ok?

Reply:
sure